lib/castle_web/views/api/episode_view.ex

```diff
@@ -52,7 +52,7 @@ defmodule CastleWeb.API.EpisodeView do
        templated: true,
       },
       "prx:totals": %{
-        href: api_episode_total_path(conn, :index, episode.id) <> "{?from,to}",
+        href: api_episode_total_path(conn, :index, episode.id) <> "{?interval,from,to,group}",
```

Comment:
The `interval` isn't actually used in that Totals controller - it only needs the `group`

Reply:
ah... ok 👍